apps/test/unit/tutorialExplorer/filterGroupOrgNamesTest.jsx

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import {shallow} from 'enzyme';
+import sinon from 'sinon';
+import {expect} from '../../util/configuredChai';
+import FilterGroupOrgNames from '@cdo/apps/tutorialExplorer/filterGroupOrgNames';
+import FilterGroupContainer from '@cdo/apps/tutorialExplorer/filterGroupContainer';
+import i18n from '@cdo/tutorialExplorer/locale';
+
+const TEST_ORG_NAME = 'Code Studio';
+const ORG_1 = 'Hogwarts School';
+const ORG_2 = 'Miskatonic University';
+const ORG_3 = 'Starfleet Academy';
+const TEST_UNIQUE_ORG_NAMES = [
+  ORG_1,
+  ORG_2,
+  ORG_3
+];
```

Comment:
I might be confused about what filter means in this context, but should their be a test with a list of org names containing a duplicate?

Reply:
Great question.  I think here "filter" refers to the user's controls for filtering the visible set of tutorials.  The `FilterGroupOrgNames` component takes a `uniqueOrgNames` prop, and assumes it already has a set of unique names.

We should have a test checking that we end up with unique org names though.  That logic lives up a few layers, probably near [this code in tutorialExplorer.js](https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/tutorialExplorer/tutorialExplorer.js#L219-L221), which might be [partially covered here](https://github.com/code-dot-org/code-dot-org/blob/staging/apps/test/unit/tutorialExplorer/TutorialExplorerTest.js#L168-L183) but could probably use a little more checking of edge cases.